app/modules/code_provider/github/github_service.py

```
@@ -455,667 +455,667 @@ def _parse_link_header(self, link_header: str) -> Dict[str, str]:
+                    parsed_url.netloc,
+                    parsed_url.path,
+                    parsed_url.params,
+                    new_query,
+                    parsed_url.fragment,
+                )
+            )
+
+            if len(first_page_url) > 2000:
+                return None, None
+            return first_page_url, parsed_url
+        except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)